features/admin.users.v1/components/wizard/bulk-import-user-wizard.tsx

```diff
@@ -707,7 +708,7 @@ export const BulkImportUserWizard: FunctionComponent<BulkImportUserInterface> =
                     }
 
                     dataObj[RequiredBulkUserImportAttributes.USERNAME] = selectedUserStore &&
-                    selectedUserStore.toLowerCase() !== userstoresConfig.primaryUserstoreName.toLowerCase()
```

Comment:
This improvement was done to let users configure custom userstores as the primary userstore. Changing it back to using hardcoded primary userstore (PRIMARY) will cause issues for IS users.

cc: @DilshanSenarath

Reply:
If the primary user store is renamed (e.g., to 'MAIN'), the user store name must be prepended to the username.

Reply:
No, we need to use the constant `PRIMARY` domain here because, even if the customer changes the domain, it must still be appended to the username. Domain appending can only be ignored when the domain is `PRIMARY`.

Reply:
> No, we need to use the constant `PRIMARY` domain here because, even if the customer changes the domain, it must still be appended to the username. Domain appending can only be ignored when the domain is `PRIMARY`.

Ack, hope this fact is documented somewhere, because there is no clue about that in the console.

Reply:
Ack.